### PR TITLE
Tf 01 008 communication

### PR DESCRIPTION
**Pull request by jobayeructit**
ticket: TF-01-008-Communication
[screen-capture (1).webm](https://github.com/UCT-IT/edume/assets/149924386/6d233ffe-9f58-4346-888d-ea665b3f7e14)
localhost_3000_pages_communication

### CODE REVIEW

File: src/app/pages/communication/stayInTouch/RightComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <Image
+          src={img}
+          alt=""
+          className="relative w-full rounded-md shadow-customShadow"
+        />
+        {/* <Image
+          src={calendarImg2}
+          alt=""
+          className="relative bottom-2 -left-8 shadow-customShadow w-full rounded-md"
+        /> */}
```
Comment (Rasheduluctit) [inline src/app/pages/communication/stayInTouch/RightComponent.tsx:17]: remove the comment if not necessary.

File: src/app/pages/communication/page.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      />
+      <CommonSection
+        rightComponent={<EliminateRightComponent />}
+        leftComponent={<EliminateLeftComponent />}
+        orderFirst="last"
+        orderLast="first"
+      />
+      <CommonSection
+        rightComponent={<StayInTuochRightComponent />}
+        leftComponent={<StayInTuochLefttComponent />}
```
Comment (saminsss) [inline src/app/pages/communication/page.tsx]: Spelling mistake for `StayInTuochLefttComponent`

File: src/app/pages/communication/page.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        orderFirst="first"
+        orderLast="last"
+      />
+      <CommonSection
+        rightComponent={<AutomatedRightComponent />}
+        leftComponent={<AutomatedLefttComponent />}
```
Comment (saminsss) [inline src/app/pages/communication/page.tsx]: Spelling mistake `AutomatedLefttComponent`

File: src/app/pages/communication/page.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        orderFirst="last"
+        orderLast="first"
+      />
+      <CommonSection
+        rightComponent={<AutomatedRightComponent2 />}
+        leftComponent={<AutomatedLefttComponent2 />}
```
Comment (saminsss) [inline src/app/pages/communication/page.tsx]: Spelling mistake `AutomatedLefttComponent2`

Comment (saminsss) [reply src/app/pages/communication/page.tsx]: Rename `AutomatedRightComponent2` and `AutomatedLefttComponent2` to something more distinctive

**Review by saminsss — CHANGES_REQUESTED** [review]
Left a few comments. Can merge after addressing comments

File: src/app/pages/communication/page.tsx (hunk 4 above)
Comment (jobayeructit) [reply src/app/pages/communication/page.tsx]: spelling mistake solved and renamed component name

**Pull request merged**